### PR TITLE
DOC: sample variance -> population variance (#46482)

### LINKED ISSUE #46482: DOC: sample variance -> population variance
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

https://pandas.pydata.org/docs/dev/user_guide/gotchas.html

### Documentation problem

Differences with NumPy
For [Series](https://pandas.pydata.org/docs/dev/reference/api/pandas.Series.html#pandas.Series) and [DataFrame](https://pandas.pydata.org/docs/dev/reference/api/pandas.DataFrame.html#pandas.DataFrame) objects, [var()](https://pandas.pydata.org/docs/dev/reference/api/pandas.DataFrame.var.html#pandas.DataFrame.var) normalizes by
N-1 to produce unbiased estimates of the sample variance, while NumPy’s numpy.var() normalizes by N, which measures the variance of the sample. 

"unbiased estimates of **the sample variance**" needs to be corrected to '**the population variance**'

### Suggested fix for documentation

"unbiased estimates of **the sample variance**" needs to be corrected to '**the population variance**'

### PR DESCRIPTION
**Pull request by kwhkim**
- [x] closes #46482 (Replace xxxx with the Github issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: doc/source/user_guide/gotchas.rst [hunk 1]
```diff
@@ -367,7 +367,7 @@ integer arrays to floating when NAs must be introduced.
 Differences with NumPy
 ----------------------
 For :class:`Series` and :class:`DataFrame` objects, :meth:`~DataFrame.var` normalizes by
-``N-1`` to produce unbiased estimates of the sample variance, while NumPy's
+``N-1`` to produce unbiased estimates of the population variance, while NumPy's
```
Comment (jreback) [inline doc/source/user_guide/gotchas.rst:370]: why don't we also put the appropriate wiki link here as well.

Comment (kwhkim) [reply doc/source/user_guide/gotchas.rst]: sounds good. for those who are confused with sample and population variance

**Review by mroeschke — APPROVED** [review]
LGTM. Could you merge in the main branch one more time?

**Comment by kwhkim** [comment]
@[mroeschke](https://github.com/mroeschke) sorry i dont understand what you mean.... you mean another pull request?

**Comment by jreback** [comment]
> @[mroeschke](https://github.com/mroeschke) sorry i dont understand what you mean.... you mean another pull request?

no what you did is fine

**Review by jreback — APPROVED** [review]

File: doc/source/user_guide/indexing.rst [hunk 2]
```diff
@@ -1553,9 +1553,7 @@ Index objects
 -------------
 
 The pandas :class:`~pandas.Index` class and its subclasses can be viewed as
-implementing an *ordered multiset*. Duplicates are allowed. However, if you try
-to convert an :class:`~pandas.Index` object with duplicate entries into a
-``set``, an exception will be raised.
+implementing an *ordered multiset*. Duplicates are allowed.
```
Comment (mroeschke) [inline doc/source/user_guide/indexing.rst:1556]: Why did this change?

Comment (kwhkim) [reply doc/source/user_guide/indexing.rst:1556]: @mroeschke sorry, this is for https://github.com/pandas-dev/pandas/issues/46440. I thought the comment with number in the parenthesis like (#....) would be sufficient for pointing to a specific issue but it must not. I do not know how to move this PR... :( would you help me here?

Comment (mroeschke) [reply doc/source/user_guide/indexing.rst:1556]: You can just revert this change like

```
git revert b1521ea
git push origin <your_branch>
```

Comment (kwhkim) [reply doc/source/user_guide/indexing.rst:1556]: @moreschke do you mean [b4f97fe](https://github.com/pandas-dev/pandas/pull/46711/commits/b4f97feacc4821a711d01a1f5aa58a2490a81cc2)? And how do I PR to a specific issue?

Comment (kwhkim) [reply doc/source/user_guide/indexing.rst:1556]: oh i got it https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Comment by jreback** [comment]
thanks @kwhkim

**Pull request merged**